addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2871,7 +2902,9 @@ options.registry.ConditionalVisibility = options.Class.extend({
      * @override
      */
     async onTargetHide() {
-        this.$target[0].classList.add('o_conditional_hidden');
+        if (this.$target[0].classList.contains('o_snippet_invisible')) {
+            this.$target[0].classList.add('o_conditional_hidden');
+        }
```

Comment:
Why this change ?

Reply:
This is indeed not needed anymore - it was related to an intermediate development state, sorry.